gateone/applications/terminal/plugins/ssh/scripts/ssh_connect.py

```diff
@@ -895,6 +906,7 @@ def main():
                 identities=identities,
                 additional_args=options.additional_args,
                 socket=options.socket,
+		config=options.config,
```

Comment:
Same as above: The indentation doesn't seem to be correct here. Try running `flake8` over this file.